test/integration/api_key.bats

```diff
@@ -1,20 +1,12 @@
 setup_file() {
-    kubectl create namespace qdrant-helm-integration
-    kubectl create serviceaccount default -n qdrant-helm-integration || true
-    helm install qdrant charts/qdrant --set apiKey=foobar -n qdrant-helm-integration --wait
+    helm upgrade --install qdrant charts/qdrant --set apiKey=foobar -n qdrant-helm-integration --wait
```

Comment:
do we need --create-namespace, as the creation of the qdrant-helm-integration namespace was removed?

Reply:
The namespace is created at the beginning of the test-suite now: https://github.com/qdrant/qdrant-helm/pull/79/files#diff-1e6607aa24d2be0c0462a6dbc6ee1c08de3e02f6079d4a89eda707a112ae96c3R6